plugins/nodes/src/js/pages/nodes-overview/HostsPageContent.js

```diff
@@ -154,6 +156,34 @@ class HostsPageContent extends React.Component {
       </div>
     );
   }
+  isGridView() {
+    return this.props.location.pathname.includes("grid");
```

Comment:
We start scattering the logic. How would you feel passing `isGridActive` as a prop down from `NodesOverview`?

Reply:
hm, I dont think it is a good idea. in #3117 we will update this so that NodesOverview holds Agents and Masters Tabs… I dont think it is a good idea to have this in `NodesOverview` then.